modules/Data_Cleaning/lab/Data_Cleaning_Lab.Rmd

```diff
@@ -126,11 +126,14 @@ Check to see how many values `gender` has for each category (hint: use `count`).
 
 6. Make a new tibble of `BloodType` called `Bloodtype_split` that splits the `type` variable into two called `blood_type` and  `Rhfactor`. Note: periods are special characters that generally are interpreted as wild cards thus we need `\\` to tell R that we want it to be interpreted as a period.
```

Comment:
```suggestion
6. Make a new tibble of `BloodType` called `Bloodtype_split` that splits the `type` variable into two called `blood_type` and  `Rhfactor`. Note: periods are special characters that generally are interpreted as wild cards thus we need `\\.` instead of simply "." to tell R that we want it to be interpreted as a period.
```

Reply:
making the lab file match the key